### PR TITLE
Adds Asitava to PSLab team

### PR DESCRIPTION
**Pull request by asitava1998**
Adds asitava1998 to PSLab Team

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -296,6 +296,21 @@ <h3 id="praveen">Praveen Patil</h3>
                         </div>
                     </div>
                     <hr>
+                    <!-- Developer Three -->
+                    <div class="row">
+                        <div class="col-lg-4 col-sm-6 text-center">
+                            <img class="img-circle img-responsive img-center" src="img/asitavas.JPG" alt="">
+                            <h3>
+                                <small><a href="https://github.com/cloudypadmal">Asitava Sarkar</a></small>
```
Comment (viveksb007) [inline index.html]: Change github profile link

**Review by viveksb007 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (asitava1998) [reply index.html]: Silly mistake

File: index.html [hunk 2]
```diff
@@ -296,6 +296,21 @@ <h3 id="praveen">Praveen Patil</h3>
...
+                    <!-- Developer Three -->
+                    <div class="row">
+                        <div class="col-lg-4 col-sm-6 text-center">
+                            <img class="img-circle img-responsive img-center" src="img/asitavas.JPG" alt="">
+                            <h3>
+                                <small><a href="https://github.com/asitava1998">Asitava Sarkar</a></small>
+                            </h3>
+                        </div>
+                        <div class="col-lg-8">
+                            <h3 id="padmal">Asitava Sarkar</h3>
```
Comment (CloudyPadmal) [inline index.html]: Id should also change!

Comment (asitava1998) [reply index.html]: I just copied the format verbatim from your PR. So, while changing, some things were missed out. Sorry for that.

**Review by CloudyPadmal — APPROVED** [review]

**Review by viveksb007 — APPROVED** [review]

**Comment by mariobehling** [comment]
Please resolve conflicts. Cheers

**Comment by asitava1998** [comment]
@mariobehling  Resolved the conflicts

**Pull request merged**